src/v2/Apps/Order/Routes/__tests__/Shipping.jest.tsx

```diff
@@ -3,9 +3,11 @@ import { cloneDeep } from "lodash"
 
 import {
   UntouchedBuyOrder,
-  UntouchedBuyOrderWithArtaEnabled,
```

Comment:
Tests and Fixtures could use a refactor, I'd argue to do this as a fast follow. This entire `Shipping` file and specs are intense and contain a lot of logic